source/apiVolontaria/apiVolontaria/tests/tests_model_TemporaryToken.py

```diff
@@ -0,0 +1,61 @@
+from rest_framework.test import APIClient
+
+from rest_framework.test import APITestCase
+
+from apiVolontaria.models import TemporaryToken
+from apiVolontaria.factories import UserFactory
+from django.utils import timezone
+
+
+class TemporaryTokenTests(APITestCase):
+
+    def setUp(self):
+        self.user = UserFactory()
+
+    def test_expired_property_false(self):
+        """
+        Ensure expired() return False when the token is not expire
+        """
+        token = TemporaryToken.objects.create(
+            user=self.user
+        )
+
+        token.expires = timezone.now() + timezone.timedelta(
+            minutes=100
+        )
+
+        self.assertEquals(token.expired, False)
+
+    def test_expired_property_true(self):
+        """
+        Ensure expired() return True when the token is expire
+        """
+        token = TemporaryToken.objects.create(
+            user=self.user
+        )
+
+        token.expires = timezone.now()
+
+        # It's already not egal because time is
+        # passed since the last line of code
+        self.assertEquals(True, token.expired)
+
+    def test_expire_function_true(self):
+        """
+        Ensure expire() put the token expired
+        """
+        token = TemporaryToken.objects.create(
+            user=self.user
+        )
+
+        token.expires = timezone.now() + timezone.timedelta(
+            minutes=100
+        )
+
+        # The token is not expire, it rest 100 minutes
```

Comment:
The token is not expired, 100 minutes remain

Je sais pas si ce commentaire est vraiment nécessaire

Reply:
Des commentaires c'est jamais perdu, ca aide a la comprehension global :)